network-api/networkapi/mozfest/factory.py

```diff
@@ -66,10 +66,7 @@ def generate(seed):
     except MozfestHomepage.DoesNotExist:
         print('Generating a Homepage')
         # If this is a review app, make the root of the mozfest site a child of the main site
```

Comment:
we'll probably want to remove this comment too, then =)

Reply:
Good catch. Removed!